WordPress/Classes/ViewRelated/Suggestions/SuggestionsTableView.m

```diff
@@ -29,7 +32,7 @@ - (instancetype)initWithSiteID:(NSNumber *)siteID
         _suggestions = [[SuggestionService sharedInstance] suggestionsForSiteID:_siteID];
         _searchText = @"";
         _searchResults = [[NSMutableArray alloc] init];
-        
+        [self setUseTransparentHeader:NO];
```

Comment:
Good catch!  Thank you :) Fixed in https://github.com/wordpress-mobile/WordPress-iOS/commit/780f001fd4eda3204523994680c2713db811c0ee